website/docs/gettingstarted/safety-guidelines.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 2
+---
+
+# Safety guidelines
+
+:::caution
```

Comment:
From Docusaurus 3.0, the `:::caution` admonition is deprecated.
How about using the `warning` instead of it?
https://docusaurus.io/blog/preparing-your-site-for-docusaurus-v3#admonition-warning
```suggestion
:::warning
```